content/en/docs/concepts/architecture/nodes.md

```diff
@@ -237,17 +237,16 @@ checks the state of each node every `--node-monitor-period` seconds.
 In versions of Kubernetes prior to 1.13, NodeStatus is the heartbeat from the
 node. Node lease feature is enabled by default since 1.14 as a beta feature
 (feature gate `NodeLease`, [KEP-0009](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/0009-node-heartbeat.md)).
-When node lease feature is enabled, each node has an associated `Lease` object in
-`kube-node-lease` namespace that is renewed by the node periodically, and both
-NodeStatus and node lease are treated as heartbeats from the node. Node leases
-are renewed frequently while NodeStatus is reported from node to master only
-when there is some change or enough time has passed (default is 1 minute, which
-is longer than the default timeout of 40 seconds for unreachable nodes). Since
-node lease is much more lightweight than NodeStatus, this feature makes node
-heartbeat significantly cheaper from both scalability and performance
-perspectives.
-
-In most cases, node controller limits the eviction rate to
+
+When NodeLease is enabled, each Node has an associated Lease object in the
+`kube-node-lease` namespace that the kubelet periodically renews. The control
```

Comment:
@sftim, how is this change related to PR #17189?

Reply:
PR #17189 is about promoting NodeLease to GA. Both cover the same text and if this one lands first I was expecting to help out with tweaks / a rebase for #17189

Reply:
Yes, that is true, however, the GA related content is nearly identical and is intended for the upcoming 1.17 release.
This change will definitely interfere with that merge.

Reply:
Maybe the easy option is to leave this until v1.17 is out?